Tasks/split.js

```diff
@@ -1,10 +1,12 @@
 // Split an array into two parts
+'use strict';
 
-let split_Array = (index, array) => {
-  begin = array.slice(0, index);
-  const len = array.length;
-  array = array.slice(index, len);
-  return [begin, array];
+const splitArray = (divider, array) => {
+  const START = 0;
+  const firstArray = array.slice(START, divider);
```

Comment:
It's ok to have 0 and 1 numbers hardcoded in most cases
```suggestion
  const firstArray = array.slice(0, divider);
```